.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     ],
     'react/prop-types': 'off',
     'react/react-in-jsx-scope': 'off',
+    'react/display-name': 'off', // Disable to write display name for forwardRef
```

Comment:
Where did you use this? If it's just one place you need this, it's better to just disable that single line.
```ts
// eslint-disable-next-line react/display-name
```

Reply:
Ok. :)